src/components/notification/__tests__/NotificationBadge.tsx

```diff
@@ -0,0 +1,84 @@
+import * as React from 'react';
+
+import { mount } from 'enzyme';
+import NotificationBadge from '../NotificationBadge';
+import { AcademyNotification } from '../notificationShape';
+
+const studentNotifications: AcademyNotification[] = [
+  {
+    id: 1,
+    type: 'new',
+    assessment_id: 1,
+    assessment_type: 'Mission',
+    assesssment_title: 'The Secret to Streams'
+  },
+  {
+    id: 2,
+    type: 'graded',
+    assessment_id: 1,
+    assessment_type: 'Mission',
+    assesssment_title: 'The Secret to Streams'
+  },
+  {
+    id: 3,
+    type: 'autograded',
+    assessment_id: 1,
+    assessment_type: 'Mission',
+    assesssment_title: 'The Secret to Streams'
+  },
+  {
+    id: 4,
+    type: 'unsubmitted',
+    assessment_id: 1,
+    assessment_type: 'Mission',
+    assesssment_title: 'The Secret to Streams'
+  }
+];
+
+const avengerNotifications: AcademyNotification[] = [
+  {
+    id: 1,
+    type: 'submitted',
+    assessment_id: 1,
+    assessment_type: 'Mission',
+    assesssment_title: 'The Secret to Streams',
+    submission_id: 1
+  }
+];
+
+const handleAcknowledgeNotification = (ids: number[]) => null;
+
+describe('Badge', () => {
```

Comment:
The snapshots for these tests don't even give any useful information. How are these tests going to be useful?

For example, I cannot even see the notifications data (since they are prob triggered on popover)

Reply:
I understand. I'm not sure how to test the popover so these are very basic tests